<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="newusers-MacBook-Pro.local" ignored="2" name="HubSpot Fuctional Regression Test" tests="2" failures="1" timestamp="2020-07-25T02:12:27 EDT" time="32.038" errors="0">
  <testcase name="verifyPageTitletest" time="5.278" classname="com.qa.hubspot.tests.LoginPageTest"/>
  <testcase name="verifySignUpLinkTest" time="5.413" classname="com.qa.hubspot.tests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.hubspot.tests.LoginPageTest.verifySignUpLinkTest(LoginPageTest.java:76)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifySignUpLinkTest -->
  <testcase name="login_invalidTestCase" time="0.0" classname="com.qa.hubspot.tests.LoginPageTest">
    <ignored/>
  </testcase> <!-- login_invalidTestCase -->
  <testcase name="loginTest" time="0.0" classname="com.qa.hubspot.tests.LoginPageTest">
    <ignored/>
  </testcase> <!-- loginTest -->
</testsuite> <!-- HubSpot Fuctional Regression Test -->
